src/components/agencies/view-agency/view-agency.component.js

```diff
@@ -30,16 +41,36 @@ class ViewAgency extends Component {
     });
   };
 
+  goEditUser = (id) => {
+    //TODO: Use router!
```

Comment:
Can you say more about this "TODO"? what should be done?

Reply:
Sorry I forgot to remove the comment, I have added logic  `goToPage(EDIT_USER_PAGE, id);` under the function, which navigates to edit user page.